include/solvers/gls_sharp_navier_stokes.h

```diff
@@ -67,33 +67,43 @@ class GLSSharpNavierStokesSolver : public GLSNavierStokesSolver<dim>
   // BB - TODO The particles structure should be refactored to use a small class
   // to store the information or a struct instead of just using a vector where
   // the things are hardcoded within.
+
+  // Defined the particle structure and value based on the parameter file.
```

Comment:
Fix the comments here. There is still this BB- TODO
Additionnaly, use present tense.